Classes/Command/ImportCommand.php

```diff
@@ -51,6 +52,7 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        Bootstrap::initializeBackendAuthentication();
```

Comment:
Is this really needed?

Reply:
In my Project context this did not work without initializing the user